core/src/main/resources/defaults/clustertemplates/7.2.15/azure/rt-datamart.json

```diff
@@ -83,8 +83,8 @@
         "template": {
           "attachedVolumes": [
             {
-              "count": 7,
-              "size": 1000,
+              "count": 0,
```

Comment:
I think we wanted to use the ephemeral (temp SSD) storage in this case.  Is specifying a value of zero here the correct way to specify ephemeral use in an Azure template (just asking because it's not intuitive and syntax is different from the AWS templates where we explicitly use the ephemeral label for type)?

Same question applies for 7.2.16 and 7.2.16 versions of the template below.

Reply:
For this, I have followed the pattern which we have used for the datamart template for azure and currently for azure, we do not support ephemeral volumes as well. So in DISTX-681 as well for azure side changes we had made the volume count to 0 so that only the ephemeral or the local volumes are used.

Reply:
OK sounds good.  We should verify via testing that Impala ends up with the correct configuration to use the Azure ephemeral volumes in this case, but your explanation makes sense.